README.md

```diff
@@ -1,2 +1,27 @@
-# The-Async-Force-Ep-2
-Async Exercise using Interactive Client side XHR accessing the swapi
+# The-Async-Force-Ep-1
+An **async** Exercise using **client side** XHR accessing the swapi API at https://swapi.co
+
+
+
+## Excercise
+*Remember to commit after each step.*
+
+1. Make a `/js` folder and create an `app.js` file in it.
+1. In the `index.html` file, load `app.js` via script tags.
+1. Use live-reload to serve up the files to your browser.
+1. Construct your XHR requests in the `app.js` file by scraping the value from the input field.
+1. A request to the SWAPI api should be initiated when the button is clicked.
+  Information from the data that comes back from the API should be displayed in
+  the body of the html.  Additional html elements should be created to properly
+  display and format the data in a presentable way.
+1. Think of an elegant way of handling errors.  For example, if the SWAPI API returns an error due to incorrect an incorrect URL fragment.
```

Comment:
Display any xhr errors in the dom, so the user can see.
for example, if a user inputs `99999` as an id, or `potato`
display the error on the page